### PR TITLE
Add article "Using ESP BSP with DevKits"

### PR DESCRIPTION
**Pull request by espzav**
## Description

Article about Generic BSP

## Checklist

Before submitting a Pull Request, please ensure the following:

- [ ] 🚨 This PR does not introduce breaking changes.
- [ ] All CI checks (GH Actions) pass.
- [ ] Documentation is updated as needed.
- [ ] Tests are updated or added as necessary.
- [ ] Code is well-commented, especially in complex areas.
- [ ] Git history is clean — commits are squashed to the minimum necessary.

### CODE REVIEW

**Comment by espzav** [comment]
@igrr @tore-espressif PTAL

File: content/blog/using-esp-bsp-with-devkits/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Using ESP BSP with DevKits"
+date: 2024-09-17
+showAuthor: false
+tags: ["Embedded Systems", "ESP32", "ESP32-S2", "ESP32-S3", "ESP32-C2", "Espressif", "BSP", "DevKit"]
+---
+
+## Introduction
+
+The BSP (Board Support Package) from Espressif is ready to use package for selected boards (usually for Espressif's and MStack boards). This package contains main initialization functions for features, which the board contains. For example **display** and **touch** initialization, **file system** initialization, **audio** if available, **LEDs** and **buttons**. **This is the easy way, how to start developing with your board.** Except initialization functions, there are examples for selected board too.
```
[the remark this answers is not in the document]
Comment (espzav) [reply content/blog/using-esp-bsp-with-devkits/index.md]: Fixed

File: content/blog/using-esp-bsp-with-devkits/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Using ESP BSP with DevKits"
+date: 2024-09-17
+showAuthor: false
+tags: ["Embedded Systems", "ESP32", "ESP32-S2", "ESP32-S3", "ESP32-C2", "Espressif", "BSP", "DevKit"]
```
[the remark this answers is not in the document]
Comment (espzav) [reply content/blog/using-esp-bsp-with-devkits/index.md:7]: Added.

**Comment by espzav** [comment]
@pedrominatel Thank you for the review :-)

**Comment by pedrominatel** [comment]
Hi @espzav, please look at the final review and the featured image.
Please note that I've changed the date to the future, so if you run `hugo serve` the article will not be visible, only if you change to a past date. As soon as you finish your final review, I'll merge and the article will be visible on the selected date.

Thank you.

**Review by pedrominatel — APPROVED** [review]

**Comment by espzav** [comment]
Hi @pedrominatel, Thank you very much for changes and for the featured image! You can merge it :-)

**Pull request merged**